eZ/Publish/API/Repository/Tests/BaseTest.php

```diff
@@ -11,6 +11,7 @@
 use Doctrine\DBAL\Connection;
 use eZ\Publish\API\Repository\Exceptions\ContentFieldValidationException;
 use eZ\Publish\API\Repository\Tests\PHPUnitConstraint\ValidationErrorOccurs as PHPUnitConstraintValidationErrorOccurs;
+use eZ\Publish\API\Repository\Values\Content\Language;
 use eZ\Publish\API\Repository\Values\Content\Location;
```

Comment:
Side stuff, seems `Location` is unused. Fixer is confused by 
```
$this->generateId('location', 4);
```
so it didn't report it :grinning: